Jenkinsfile

```diff
@@ -0,0 +1,112 @@
+#!groovy
+
+/*
+ * Copyright © 2016 IBM Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+stage('Build') {
+    // Checkout, build and assemble the source and doc
+    node('android-compile') {
```

Comment:
Following on from our discussion, yes, we can do this anywhere for now because of the old Android level available from maven for us to compile against - so I'll remove the node label restriction.
At some point when we update the minimum Android level we'll need some changes.